src/processVesselDataset.py

```diff
@@ -70,8 +72,12 @@
 plt.tight_layout()
 plt.show()
 
-# fig.savefig('map.png', dpi=400, bbox_inches='tight',
-#             pad_inches=0)
+fig.savefig('map.png', dpi=400, bbox_inches='tight',
```

Comment:
Also save the figure with the trajectories when running this script.

Reply:
I set the dpi to 400 last time, but that is waaaay too large for the program. I manually made the map smaller with photoshop, but if you use something like 100 the pixel width and height will already be much less.